{"metadata":{"role":"collectionGroup","title":"LocalizedError Implementations","modules":[{"name":"Network"}],"roleHeading":"API Collection"},"topicSections":[{"anchor":"Instance-Properties","identifiers":["doc:\/\/Network\/documentation\/Network\/NetworkError\/failureReason","doc:\/\/Network\/documentation\/Network\/NetworkError\/helpAnchor","doc:\/\/Network\/documentation\/Network\/NetworkError\/recoverySuggestion"],"title":"Instance Properties","generated":true}],"variants":[{"paths":["\/documentation\/network\/networkerror\/localizederror-implementations"],"traits":[{"interfaceLanguage":"swift"}]}],"hierarchy":{"paths":[["doc:\/\/Network\/documentation\/Network","doc:\/\/Network\/documentation\/Network\/NetworkError"]]},"schemaVersion":{"minor":3,"patch":0,"major":0},"sections":[],"kind":"article","identifier":{"url":"doc:\/\/Network\/documentation\/Network\/NetworkError\/LocalizedError-Implementations","interfaceLanguage":"swift"},"references":{"doc://Network/documentation/Network/NetworkError":{"identifier":"doc:\/\/Network\/documentation\/Network\/NetworkError","type":"topic","title":"NetworkError","abstract":[{"text":"Errors that can occur during network operations.","type":"text"}],"url":"\/documentation\/network\/networkerror","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"NetworkError","kind":"identifier"}],"navigatorTitle":[{"text":"NetworkError","kind":"identifier"}],"kind":"symbol","role":"symbol"},"doc://Network/documentation/Network/NetworkError/helpAnchor":{"type":"topic","url":"\/documentation\/network\/networkerror\/helpanchor","kind":"symbol","abstract":[],"title":"helpAnchor","identifier":"doc:\/\/Network\/documentation\/Network\/NetworkError\/helpAnchor","role":"symbol","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"helpAnchor","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"},{"text":"?","kind":"text"}]},"doc://Network/documentation/Network/NetworkError/recoverySuggestion":{"kind":"symbol","role":"symbol","url":"\/documentation\/network\/networkerror\/recoverysuggestion","title":"recoverySuggestion","type":"topic","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"recoverySuggestion","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"},{"text":"?","kind":"text"}],"abstract":[],"identifier":"doc:\/\/Network\/documentation\/Network\/NetworkError\/recoverySuggestion"},"doc://Network/documentation/Network":{"identifier":"doc:\/\/Network\/documentation\/Network","url":"\/documentation\/network","abstract":[],"role":"collection","type":"topic","title":"Network","kind":"symbol"},"doc://Network/documentation/Network/NetworkError/failureReason":{"kind":"symbol","role":"symbol","url":"\/documentation\/network\/networkerror\/failurereason","title":"failureReason","type":"topic","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"failureReason","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"},{"text":"?","kind":"text"}],"abstract":[],"identifier":"doc:\/\/Network\/documentation\/Network\/NetworkError\/failureReason"}}}